### PR TITLE
Add Siler, Siler-Swoole and Yii-Swoole

### PR DESCRIPTION
**Pull request by RicardoSette**
Hi @waghanza,

from #2491 , #2492 and #2490

### CODE REVIEW

File: php/siler-swoole/composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+      "leocavalcante/siler": "dev-master"
```
Comment (waghanza) [inline php/siler-swoole/composer.json:3]: could you use `~1.7.2` instead of `dev-master` ?

File: php/siler/composer.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "require": {
+    "leocavalcante/siler": "dev-master"
```
Comment (waghanza) [inline php/siler/composer.json:3]: could you use `~1.7.2` instead of `dev-master` ?

File: php/siler-swoole/config.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+framework:
+  github: leocavalcante/siler
```
Comment (waghanza) [inline php/siler-swoole/config.yaml:2]: this line could be removed (having `website` give the same kind of info)

File: php/siler/config.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+framework:
+  github: leocavalcante/siler
```
Comment (waghanza) [inline php/siler/config.yaml:2]: this line could be removed (having `website` give the same kind of info)

File: php/siler-swoole/composer.json (hunk 1 above)
Comment (RicardoSette) [reply php/siler-swoole/composer.json:3]: @leocavalcante it is ok?

Comment (leocavalcante) [reply php/siler-swoole/composer.json:3]: @RicardoSette, no problem. The latest stable release supports features used in the test.

Comment (waghanza) [reply php/siler-swoole/composer.json:3]: @RicardoSette @leocavalcante version will be in-sync :heart: Thanks to `dependabot` and `renovate`

Comment (leocavalcante) [reply php/siler-swoole/composer.json:3]: @waghanza how cool! Just added to Siler itself, thanks for the hint.

**Pull request closed without merging**